test/images/nettest/nettest.go

```diff
@@ -278,7 +280,8 @@ func getWebserverEndpoints(client clientset.Interface) sets.String {
 	for _, ss := range endpoints.Subsets {
 		for _, a := range ss.Addresses {
 			for _, p := range ss.Ports {
-				eps.Insert(fmt.Sprintf("http://%s:%d", a.IP, p.Port))
+				ipPort := net.JoinHostPort(a.IP, strconv.Itoa(int(p.Port)))
```

Comment:
As we already do `int(p.Port)` just pass this directly to `net.JoinHostPort()`. The `strconv.Itoa()` is not necessary.

Reply:
p.Port in this case is an int32. The int(p.Port) is an intermediate conversion from int32 to int, since strconv.Itoa() won't take int32 directly. I could use fmt.Sprint(p.Port), if the double conversion is misleading.